Amazon Scraper/scrape.py

```diff
@@ -0,0 +1,49 @@
+import csv
+
```

Comment:
This is not what I meant when I said "Give appropriate spaces in the program to differentiate different sections of the code". Club all the import statements. Appropriate spacing in sections means if 3 lines of code do scrapping club them if 4 lines of code save a file club them and between these sections, give a space. You have done for the rest of the code but not for the initial codes